File: 1_Período/Lista 10/C/L10_Q03.c
#include <stdio.h>

int main(void){
	
	int v, vmaior, vmenor, r= 0;
	
	printf("Informe um valor inteiro: ");
	scanf("%d", &v);
	
	vmaior = v;
	vmenor = v;
	
	while( v >= 0){
		if(v > vmaior)
			vmaior = v;
		if(v < vmenor)
			vmenor = v;
		
		r += v;
		
		printf("Informe um valor inteiro: ");
		scanf("%d", &v);
	}
	
	printf("Maior valor : %d\nMenor valor: %d\nSoma dos valores: %d", vmaior, vmenor, r);
	
	return 0;
}

File: 1_Periodo/Lista 10/Python/L10_Q03.py
soma = 0
menor = float('inf')
maior = float('-inf')

for i in range(10):
    numero = float(input(f"Informe o {i+1}º numero: "))
    soma +=numero

    if numero < menor: menor = numero

    if numero > maior: maior = numero

print()
print("Maior valor: ", maior)
print("Menor valor: ", menor)
print("Soma dos valores: ", soma)
print()